test/objects.js

```diff
@@ -775,11 +775,12 @@
     assert.ok(_.has(obj, 'func'), 'has() works for functions too.');
     obj.hasOwnProperty = null;
     assert.ok(_.has(obj, 'foo'), 'has() works even when the hasOwnProperty method is deleted.');
-    var child = {};
-    child.prototype = obj;
+    var child = _.create(obj);
```

Comment:
Ha, fair point. You know what I mean though, shouldn't we set the proto chain manually here?